### ISSUE TITLE
Crash on `acquireTokenSilent` method

### ISSUE THREAD

**Issue by oferRounds** [vmurin/react-native-azure-auth#57]
Hi @vmurin 
Please see screenshot attached. The 
![image](https://user-images.githubusercontent.com/1477963/69623644-ee9bc900-104b-11ea-8578-a88a47cb2955.png)

The error is caught here: https://github.com/vmurin/react-native-azure-auth/blob/master/src/auth/index.js#L199

**Comment by oferRounds** [comment]
I see what the issue is – `input` is undefined, it‘s trying to get `input.scope`
![image](https://user-images.githubusercontent.com/1477963/69624197-e728ef80-104c-11ea-84b6-b76121a5ae62.png)

**Comment by oferRounds** [comment]
This issue also causes crash – when `key` is undefined
![image](https://user-images.githubusercontent.com/1477963/69625500-5f90b000-104f-11ea-8062-43c49242f11b.png)

**Comment by oferRounds** [comment]
Ok, see PR for the suggested fix (was actually on AccessTokenItem.js file)

**Comment by vmurin** [comment]
If the blue line the debugger position - the `input` and `key` can not be defined. You have to step one line further ;)

>was actually on AccessTokenItem.js file

I saw the PR. Thanks! There is really a bug, but the correct code should be

```js
        if (!rawObject || !rawObject['accessToken']) {
            return null
        }
```

I will close the PR and push this change. Thanks once more time! :)

**Comment by vmurin** [comment]
closed with 3325452

**Comment by oferRounds** [comment]
Sounds good, thanks again!
